src/trumbowyg.js

```diff
@@ -1486,6 +1486,37 @@ jQuery.trumbowyg = {
         getRangeText: function () {
             return this.range + '';
         },
+        expandRange: function () {
```

Comment:
I'm not OK to add a *huge* function to core for that :/
Need to optimize the size...

Reply:
Seems, that it takes expandes selected range till beginning and ending of paragraph.
For example, if you select the word "Ipsum" in paragraph "Lorem ipsum — dolor sit amet, consectetuer adipiscing elit", then you will get all the sentence.

But I don't understand, why do we need it. I guess, that it's a specific task for a commited user, but it without it plugin should work more correct. 

That is why:
LOC 52  `trumbowyg.expandRange();` in plugins/fontsize/trumbowyg.fontsize.js should also be remove.
>